SET_OF_STANDARDS.md

```diff
@@ -0,0 +1,50 @@
+# Programming Set of Standards
+
+## Purpose
+These are primarily guidelines on contributing ideas and code to the WeLike codebase. As well, there are other helpful insights into what is expected of us as technology industry professionals.
```

Comment:
a/WeLike/QueerGlobal/g?

Reply:
Yeah, I used a doc from a former project. I look forward to getting lots of eyes on it, to change what it all says since these are all my opinions so far.